books/All.md

```
@@ -10750,3 +10750,107 @@ def countTrapezoids(points):
     res = (total_sum * total_sum - sum_of_squares) // 2
     return res % MOD
 ```
+
+## 3709. Design Exam Scores Tracker [Medium]
+https://leetcode.com/problems/design-exam-scores-tracker/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)